x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/trigger/schedule/engine/TickerScheduleEngineTests.java

```diff
@@ -254,6 +254,29 @@ public void accept(Iterable<TriggerEvent> events) {
         assertThat(counter.get(), is(2));
     }
 
+    public void testAddOnlyWithNewSchedule() {
+        engine.start(Collections.emptySet());
+        final List<TriggerEvent> triggerEvents = new ArrayList<>();
+        engine.register(new Consumer<Iterable<TriggerEvent>>() {
+            @Override
+            public void accept(Iterable<TriggerEvent> events) {
+                events.forEach(triggerEvents::add);
+            }
+        });
```

Comment:
minor: Is the register and triggerEvents needed here ?

Reply:
indeed. removed.